src/nps.ts

```diff
@@ -41,7 +43,15 @@ export async function checkAndOpenNPSLink() {
           .add(3, 'months')
           .toISOString()
       )
-      opn(NPSFormURL, { wait: false })
+      const configClient = ToolbeltConfig.createClient()
+      const { featureFlags } = await configClient.getGlobalConfig()
+
+      if (featureFlags.FEATURE_FLAG_NEW_OPEN_PACKAGE) {
+        open(NPSFormURL, { wait: false })
+      } else {
+        opn(NPSFormURL, { wait: false })
+      } 
+      
```

Comment:
_non blocking suggestion_:
The if with `featureFlags.FEATURE_FLAG_NEW_OPEN_PACKAGE` is used in a lot of places. You can make a selector somewhere else... in the future, if the logic gets more complex (perhaps you may need to use a _switch_ IDK)
```
const openPackageSelector(withFeatureFlag, withoutFeatureFlag) {
  // Logic to get feature flags...
 if (featureFlags.FEATURE_FLAG_NEW_OPEN_PACKAGE) {
    return withFeatureFlag
  } else {
    return withoutFeatureFlag
  }
}
```

Reply:
In my point of view, the use o `Feature Flags` in complex cases need to encapsulate the login in other functions, like [here](https://github.com/vtex/cli-plugin-deploy/blob/7e936891abd5a2cf0e77bd82216bc441a67cbf9a/src/commands/deploy.ts#L24), but in this case I don't think that needed. What do you think?

Reply:
I don't think it's needed. But the suggestion is about creating a selector of the function that will be used. Something that analyses the feature flags and the possibilities of execution, and chooses one. The ideia is to simply separate the logic of the feature flag selection from the rest of the code

Reply:
Anyway, it's not a blocking comment, feel free to move on